certified-connectors/Mobsim Send SMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,133 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "MOBSIM Send SMS",
+    "description": "Communicate with your customers in an assertive and direct way using the \"MOBSIM - Send SMS\" Connector. SMS - with high delivery and opening rates - makes it possible to send promotional and transactional campaigns, alerts, reminders, notifications, invites and more, allowing for a complete experience with personalized messages and the convenience of scheduled deliveries to mobile phone users.",
```

Comment:
@jpjsantos please make the description more concise. It should describe the main purpose and value offered by your connector. More information can be found [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector).